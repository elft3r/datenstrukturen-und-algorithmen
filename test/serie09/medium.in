5
10 30 1 2 184 2 3 876 3 4 538 4 5 693 5 6 711 6 7 856 7 8 676 8 9 296 9 10 737 1 7 81 7 4 422 3 9 173 10 1 4 1 6 633 1 3 469 5 9 596 5 7 837 7 2 88 10 4 430 2 6 76 9 2 787 8 5 198 1 1 373 2 10 892 8 6 483 10 7 487 9 7 575 2 5 868 1 5 224 9 4 964
20 60 1 2 475 2 3 921 3 4 634 4 5 155 5 6 188 6 7 902 7 8 973 8 9 911 9 10 383 10 11 279 11 12 813 12 13 609 13 14 568 14 15 519 15 16 870 16 17 875 17 18 149 18 19 520 19 20 573 4 14 194 17 11 346 15 4 413 7 11 967 17 2 671 16 4 463 14 17 556 5 9 536 9 9 556 11 19 684 13 8 225 18 15 550 2 9 184 9 7 190 12 12 365 19 13 557 5 18 913 2 2 385 15 15 901 3 10 846 13 10 271 20 20 375 17 1 592 4 9 824 13 5 456 8 2 942 6 15 547 5 14 169 1 19 649 16 1 194 13 18 417 7 20 310 14 6 280 10 17 487 17 5 73 17 4 584 14 10 394 6 2 707 13 3 191 9 18 615 7 5 37
30 100 1 2 564 2 3 877 3 4 244 4 5 145 5 6 850 6 7 124 7 8 235 8 9 824 9 10 614 10 11 740 11 12 726 12 13 822 13 14 147 14 15 872 15 16 480 16 17 667 17 18 343 18 19 98 19 20 259 20 21 893 21 22 378 22 23 968 23 24 570 24 25 416 25 26 676 26 27 916 27 28 283 28 29 648 29 30 586 21 8 974 21 15 721 27 30 507 29 20 636 6 9 637 28 23 553 7 1 475 22 11 331 23 30 676 2 4 983 7 23 402 14 26 778 3 8 283 24 27 700 7 27 817 4 6 189 13 25 914 14 27 691 4 9 923 5 8 608 11 26 473 26 29 523 20 16 279 14 10 90 9 7 416 4 25 994 9 26 850 25 12 573 14 24 300 23 10 667 5 24 131 16 2 53 4 13 729 17 15 612 28 26 938 2 10 7 16 10 997 10 27 867 1 17 426 19 12 513 11 8 28 15 26 235 3 18 461 21 18 766 15 23 227 21 27 55 9 15 750 2 2 184 6 10 307 25 14 884 20 14 159 20 22 654 23 4 823 26 3 622 16 27 10 5 25 589 19 21 782 10 13 859 12 29 582 10 20 943 13 6 186 16 19 550 26 2 669 8 25 496 19 2 983 13 9 573 29 10 544 7 5 449 14 23 507 3 21 693 21 26 487
40 200 1 2 111 2 3 273 3 4 824 4 5 36 5 6 728 6 7 307 7 8 153 8 9 669 9 10 310 10 11 296 11 12 260 12 13 740 13 14 933 14 15 375 15 16 679 16 17 704 17 18 421 18 19 716 19 20 261 20 21 547 21 22 174 22 23 609 23 24 922 24 25 807 25 26 809 26 27 617 27 28 191 28 29 174 29 30 815 30 31 903 31 32 447 32 33 617 33 34 577 34 35 558 35 36 109 36 37 472 37 38 826 38 39 330 39 40 55 32 21 939 19 19 62 2 12 82 16 35 265 29 35 314 2 26 47 35 32 404 17 1 120 27 33 336 12 17 150 39 39 741 7 20 962 33 22 387 39 13 203 26 17 270 35 3 510 29 27 713 4 20 698 25 36 740 34 4 719 29 3 607 16 40 565 39 17 224 12 39 322 12 34 266 1 33 785 39 15 108 3 11 579 12 16 640 13 38 307 18 25 946 34 2 469 19 32 84 17 28 560 11 27 166 16 37 483 14 24 169 10 35 309 28 34 357 7 18 233 25 19 634 32 17 221 35 26 224 6 18 316 27 14 590 39 26 709 40 14 241 38 15 182 25 13 387 34 30 116 22 16 926 27 5 211 28 21 271 4 9 411 18 29 376 3 15 441 33 23 989 40 27 976 37 8 930 21 13 57 28 32 12 37 20 123 18 24 213 15 2 746 21 31 537 15 30 52 30 16 384 11 4 952 6 23 134 29 29 189 12 12 326 28 2 345 36 31 839 26 33 584 11 31 691 10 1 225 20 3 436 4 22 906 33 4 997 32 40 920 33 10 874 9 38 779 6 31 438 18 28 875 20 33 821 23 21 767 4 26 290 19 22 849 21 34 620 14 16 109 22 5 313 26 12 375 1 4 496 21 4 666 15 11 420 38 30 472 19 27 782 15 1 414 13 34 116 28 8 999 37 37 805 40 20 505 15 32 933 30 40 383 22 11 790 29 6 914 29 7 249 23 38 192 26 36 397 7 10 83 35 35 825 10 17 701 26 14 634 7 2 737 17 11 803 30 13 605 13 33 470 5 24 870 11 40 996 3 25 884 7 19 864 39 24 802 14 36 30 38 25 883 12 15 329 39 31 534 18 18 415 34 5 317 11 2 730 38 10 680 8 30 818 28 37 398 9 18 477 17 29 963 8 25 680 3 39 941 9 24 252 6 33 418 33 18 555 11 29 425 8 3 111 33 24 31 15 18 457 10 5 798 31 33 649 12 10 346 34 23 578 16 29 354 3 33 819 38 40 712 13 9 411 14 1 914 15 5 722 38 12 716 9 12 113 3 32 837 15 22 310 7 30 902 28 20 334 3 31 239 28 40 580
50 150 1 2 64 2 3 354 3 4 94 4 5 716 5 6 3 6 7 111 7 8 996 8 9 91 9 10 938 10 11 294 11 12 735 12 13 871 13 14 361 14 15 342 15 16 905 16 17 4 17 18 43 18 19 378 19 20 552 20 21 696 21 22 569 22 23 892 23 24 10 24 25 243 25 26 102 26 27 434 27 28 303 28 29 908 29 30 460 30 31 170 31 32 338 32 33 71 33 34 759 34 35 714 35 36 877 36 37 777 37 38 397 38 39 280 39 40 457 40 41 942 41 42 430 42 43 176 43 44 743 44 45 474 45 46 69 46 47 92 47 48 998 48 49 68 49 50 275 35 31 977 48 8 812 44 20 316 49 9 273 10 35 621 4 19 494 25 34 969 22 6 100 2 17 835 4 25 746 14 35 360 23 27 920 20 50 621 17 20 267 20 9 198 13 42 420 31 20 461 9 13 289 5 44 311 39 21 212 37 10 118 46 44 96 14 11 670 32 50 275 18 21 907 31 11 73 11 39 591 9 27 161 8 33 883 14 6 325 4 37 585 8 12 347 36 19 324 26 28 675 40 11 997 30 46 634 17 28 829 25 38 878 39 50 495 35 46 813 2 38 961 10 40 179 30 39 261 49 45 686 9 22 158 5 31 191 8 38 931 23 19 331 34 41 406 21 21 187 8 21 943 34 13 611 11 36 684 33 37 17 45 40 506 16 37 926 25 7 673 46 42 63 33 35 469 32 44 429 7 17 452 7 16 290 6 2 803 4 32 421 35 37 338 33 14 415 46 32 203 8 18 79 35 12 320 34 49 953 50 45 610 34 10 641 34 26 672 2 47 717 9 6 24 31 40 12 47 15 906 32 25 237 8 6 182 12 47 623 16 30 186 22 36 969 47 21 26 4 10 665 24 2 43 13 43 396 11 23 891 1 41 543 43 21 869 31 39 188 39 34 194 37 7 900 2 19 739 40 1 713 18 15 501 38 10 505 39 33 928 11 30 626 50 3 471 3 28 504 42 44 80
