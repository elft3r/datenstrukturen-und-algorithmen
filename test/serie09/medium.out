1562
4966
6782
5445
8212
